server/src/main/scala/org/http4s/server/middleware/Timeout.scala

```diff
@@ -13,8 +13,8 @@ object Timeout {
     * duration if the service has not yet responded.  If the timeout
     * fires, the service's response is canceled.
     *
-    * @param timeout Finite duration to wait before returning a `500
-    * Internal Server Error` response
+    * @param timeout Finite duration to wait before returning a `503
+    * Service Unavailable` response
```

Comment:
this is still wrong, it should be more in line with `the provided response`

Reply:
Changed in https://github.com/http4s/http4s/pull/3017/commits/7ea9db59afd9334f635d6c7a5ffeabf9249c162f